.github/workflows/zeebe-ci.yml

```diff
@@ -66,9 +68,15 @@ jobs:
           -D excludedGroups=$EXCLUDED_TEST_GROUPS
           -f zeebe
           verify
+      - name: Analyze Test Runs
+        id: analyze-test-run
+        if: always()
+        uses: ./.github/actions/analyze-test-runs
+        with:
+          buildOutputFilePath: ${{ env.BUILD_OUTPUT_FILE_PATH }}
       - name: Upload test artifacts
         uses: ./.github/actions/collect-test-artifacts
-        if: ${{ failure() || cancelled() }}
+        if: ${{ failure() || cancelled() || steps.analyze-test-run.outputs.flakyTests != '' }}
```

Comment:
❌ I struggle getting this one to work. The CI complains that the `BUILD_OUTPUT_FILE_PATH` doesn't exist. I've added the step to create this file, but it still complains about it. Could 1 of the reviewers have a look and see if I missed anything?

Reply:
Nvm I got it